kong/runloop/handler.lua

```diff
@@ -995,6 +1066,32 @@ return {
           log(ERR, "could not schedule timer to rebuild plugins iterator: ", err)
         end
 
+
+        if wasm.enabled() then
+          local wasm_async_opts = {
+            name = "wasm",
+            timeout = 0,
+            on_timeout = "return_true",
+          }
+
+          local function rebuild_wasm_filter_chains_timer(premature)
+            if premature then
+              return
+            end
+
+            local _, err = rebuild_wasm_state(wasm_async_opts)
+            if err then
+              log(ERR, "could not rebuild Wasm filter chains via timer: ", err)
```

Comment:
Just checking: no `return` after this error/log?

Reply:
Thanks! This is intentional; it follows the same pattern as the other `rebuild_*` functions (router, plugins iterator, etc) defined in the same context.